src/intl/messages.js

```diff
@@ -272,6 +272,15 @@ export const messages = {
   hdBoot: 'Hard Disk',
   networkBoot: 'Network (PXE)',
   cdromBoot: 'CD-ROM',
+  snapshot: 'Snapshots',
+  snapshotsTooltip: 'VM snapshots.',
+  noSnapshots: 'no snapshots',
+  newSnapshot: 'New snapshot',
+  snapshotInfo: 'All attached disks and content of memory will be included in the snapshot.',
+  addNewSnapshot: 'Add new snapshot',
+  areYouSureYouWantToDeleteSnapshot: 'Are you sure you want to delete snapshot {snapshotName}?',
+  emptySnapshotDescription: 'Please, set snapshot description.',
```

Comment:
Rephrase. Look at what oVirt has and try to use the same phrase.
Otherwise:
'Snapshot description is missing' is probably better.